### PR TITLE
No support questions

### PR DESCRIPTION
**Pull request by JesperIRL**
Added a note to refer to if people ask support questions on the mailing lists.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace

### Reviewers
 * [Iris Clark](https://openjdk.org/census#iris) (@irisclark - **Reviewer**) ⚠️ Review applies to [b2a33b35](https://git.openjdk.org/guide/pull/135/files/b2a33b35afcebc6e5057bf7b5bbc22d6ec614aa6)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/guide.git pull/135/head:pull/135` \
`$ git checkout pull/135`

Update a local copy of the PR: \
`$ git checkout pull/135` \
`$ git pull https://git.openjdk.org/guide.git pull/135/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 135`

View PR using the GUI difftool: \
`$ git pr show -t 135`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/guide/pull/135.diff">https://git.openjdk.org/guide/pull/135.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/guide/pull/135#issuecomment-2329683546)

### CODE REVIEW

**Review by irisclark — APPROVED** [review]

File: src/guide/mailing-lists.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ The mailing lists are the key communications mechanism for all OpenJDK work. All
 
 > [`mail.openjdk.org`](https://mail.openjdk.org/mailman/listinfo)
 
+:::{.note}
+Please note that OpenJDK is about **development** of the JDK, not usage. This is not a place to ask support questions. Please direct those to whoever provided your JDK build.
```
Comment (jddarcy) [inline src/guide/mailing-lists.md:14]: Nit: suggest changing to "to whoever" to "to whomever".

More substantive, I think a link to the public bug tracker where issues can be filed would be helpful here too.

Comment (JesperIRL) [reply src/guide/mailing-lists.md]: Fixed.

File: src/guide/mailing-lists.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ The mailing lists are the key communications mechanism for all OpenJDK work. All
 
 > [`mail.openjdk.org`](https://mail.openjdk.org/mailman/listinfo)
 
+:::{.note}
+Please note that OpenJDK is about **development** of the JDK, not usage. This is not a place to ask support questions. Please direct those to whomever provided your JDK build. If you think you found a bug in the JDK, see [JBS - JDK Bug System].
+:::
+
```
Comment (jddarcy) [inline src/guide/mailing-lists.md:16]: To clarify my earlier comment, I was recommending a link to be included to

https://bugreport.java.com/bugreport/

where people who do not have JBS accounts can initiate a bug being filed.

Comment (JesperIRL) [reply src/guide/mailing-lists.md:16]: The referenced section (JBS - JDK Bug System) has links to both JBS and Bugreport.

**Pull request closed without merging**